examples/api/lib/widgets/overscroll_indicator/glowing_overscroll_indicator.0.dart

```diff
@@ -14,6 +15,8 @@ class GlowingOverscrollIndicatorExampleApp extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return MaterialApp(
+      scrollBehavior: const AlwaysGlow(),
```

Comment:
This is great otherwise the glowing effect is not visible when running the example in DartPad (except when using Android).
I think it would be interesting to add a comment to mention that this is needed because this example want to demonstrate glowing effect on all platforms. And maybe add the link to https://api.flutter.dev/flutter/widgets/ScrollBehavior/buildOverscrollIndicator.html to explain that this effect is automatically available on Android.